src/ascii.rs

```diff
@@ -377,6 +378,20 @@ const BAZZITE: (&str, u16) = (
           ###################        
               ###########           ", 36);
 
+const LINUX: (&str, u16) = (
```

Comment:
What's the source for this ASCII? Or is it the same as the rest?

Reply:
Copied from neofetch.

Reply:
Add a comment then please to source it